.github/workflows/release-nightly.yml

```
@@ -11,28 +12,47 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-24.04, macos-13, macos-14, windows-2019]
-        build: [cp310, cp311, cp312]
+        os: [ubuntu-24.04]
+        build: [cp312]
     uses: ./.github/workflows/wheel.yml
     with:
       os: ${{ matrix.os }}
       build: ${{ matrix.build }}
 
   release:
-    name: Make Release on GitHub
+    name: Make Release on GitHub and pypi.anaconda.org
     needs: build_wheels
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-24.04
+    defaults:
+      run:
+        shell: bash -l {0} # required for mamba envs
     permissions:
       contents: write
     steps:
       - uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
+        with:
+          python-version: "3.10"
       - uses: actions/download-artifact@v4
         with:
           path: dist
           merge-multiple: true
-      - uses: actions/setup-python@v5
+      - name: List old dev wheels from pypi.anaconda.org
+        run: |
+          curl --silent "https://api.anaconda.org/package/scipp-nightly-wheels/scipp" | jq -r '.releases[] | select(.version | contains("dev")) | .full_name' > dev_wheels.txt
+          cat dev_wheels.txt
+      - name: Upload wheel to pypi.anaconda.org/scipp-nightly-wheels/simple
+        uses: scientific-python/upload-nightly-action@3fcd15dcbd3e25176438080493f97592a9c84e0b
```

```suggestion
        uses: scientific-python/upload-nightly-action@82396a2ed4269ba06c6b2988bb4fd568ef3c3d6b # 0.6.1
```

Thanks for your work!